specification/cognitiveservices/OpenAI.Inference/main.cadl

```diff
@@ -48,3 +58,10 @@ model Deployment {
   @key
   deploymentId: string;
 }
+
+@doc("Tokens input/total for a given API")
+model Usage {
```

Comment:
You need one per file, as they are not the same (EmbeddingUsage and CompletionUsage). CompletionUsage will have an additional `completion_tokens: int32`.